hadoop-ozone/interface-client/src/main/resources/proto.lock

```diff
@@ -4687,6 +4699,29 @@
               }
             ]
           },
+          {
+            "name": "DeleteKeyError",
```

Comment:
@xichen01 , I run the update_protolocks.sh against  the ozone-1.4.1 branch. The updated proto.lock doesn't have this DeleteKeyError message, and DeleteKeyError reference in DeleteKeysResponse.

Reply:
Sorry, it's my environment problem. 

The current patch is correct. Thanks @xichen01 .